lib/tower_cli/resources/job_template.py

```diff
@@ -58,11 +58,14 @@ class Resource(models.Resource):
     job_tags = models.Field(required=False, display=False)
     skip_tags = models.Field(required=False, display=False)
     extra_vars = models.Field(required=False, display=False)
+    ask_variables_on_launch = models.Field(
+        type=bool, required=False, display=False,
+        help_text='Prompt user for extra_vars on launch.')
```

Comment:
Note: I did add another command line option for Job Template here. That's because several fixes were applied to job launch when extra_vars prompt was enabled, and you need this in order to enable the prompt through tower-cli. You can go back and fourth between interfaces, but I would rather not leave it that way since it's a low-risk addition.